server/pbenchinacan/load_keycloak.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env -S bash -e
+
+# This scripts loads the keycloak server with following configuration
+# 1. A realm with the name pbench
+# 2. A client pbench-client under the realm pbench
+# 3. A user test_admin
+# 4. An ADMIN role under the client pbench-client
+# 5. ADMIN role assigned to the test_admin user.
+
+# At the moment the Pbench-server requires a Keycloak server with at least
+# no. 1 and no. 2 configuration specified above.
+# This scrip assumes that Keycloak server is running and does not already have
+# the configuration specified above.
+
+KEYCLOAK_HOST_PORT=${1:-"localhost:8080"}
+echo
+echo "KEYCLOAK_HOST_PORT: $KEYCLOAK_HOST_PORT"
+
+echo
+echo "Getting admin access token"
+echo "--------------------------"
+
+ADMIN_TOKEN=$(curl -s -X POST "http://$KEYCLOAK_HOST_PORT/realms/master/protocol/openid-connect/token" \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -d "username=admin" \
+  -d 'password=admin' \
+  -d 'grant_type=password' \
+  -d 'client_id=admin-cli' | jq -r '.access_token')
+
+echo "ADMIN_TOKEN=$ADMIN_TOKEN"
+echo
+
+echo "Creating realm"
+echo "--------------"
+
+curl -i -X POST "http://$KEYCLOAK_HOST_PORT/admin/realms" \
+  -H "Authorization: Bearer $ADMIN_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{"realm": "pbench", "enabled": true}'
+
+echo "Creating client"
+echo "---------------"
+
+CLIENT_CONF=$(curl -si -X POST "http://$KEYCLOAK_HOST_PORT/admin/realms/pbench/clients" \
+  -H "Authorization: Bearer $ADMIN_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{"clientId": "pbench-client", "directAccessGrantsEnabled": true, "serviceAccountsEnabled": true, "redirectUris": ["http://localhost:8080/*"]}')
```

Comment:
So the authentication request is allowed to specify any path under `http://localhost:8080`? E.g., `http://localhost:8080/dashboard`? (And does this work if the dashboard authorizes specifying `http://intlab-002.[...]:8080/dashboard/overview`??

Reply:
The `"redirectUris"` value should almost certainly be a parameter to this script, since (AFAICT) we cannot predict here where the Dashboard code will be served from.